src/main/java/com/educare/unitylend/service/BorrowRequestService.java

```diff
@@ -0,0 +1,15 @@
+package com.educare.unitylend.service;
+
+import com.educare.unitylend.Exception.ServiceException;
+import com.educare.unitylend.model.BorrowRequest;
+
+import java.util.List;
+
+public interface BorrowRequestService {
+
+    List<String> getRequestedCommunitiesByUserId(String userId) throws ServiceException;
+
+    boolean raiseBorrowRequestByUserId(String userId, BorrowRequest borrowRequest) throws ServiceException;
```

Comment:
1. The function name should be `validateBorrowRequest`.
2. I don't think that there is any need to pass userId as `BorrowRequest ` object would already be having the borrower details.